src/index.html

```diff
@@ -7,6 +7,361 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home" class="logoLink">
+        <img
+          src="/images/logo.png"
+          alt="logo"
+          class="logo"
```

Comment:
class to <a> and class to <img>. I think <img> should be an element of <a> block